guides/micronaut-metrics-oci/src/main/resources/application.yml

```
@@ -0,0 +1,58 @@
+micronaut:
+  application:
+    name: micronautguide
+---
+#tag::http-services[]
+micronaut:
+  http:
+    services:
+      kucoin: # <1>
+        urls:
+          - "https://api.kucoin.com" # <2>
+#end::http-services[]
+---
+#tag::metrics[]
+micronaut:
+  metrics:
+    enabled: true # <1>
+    binders: # <2>
+      files:
+        enabled: true
+      jdbc:
+        enabled: true
+      jvm:
+        enabled: true
+      logback:
+        enabled: true
+      processor:
+        enabled: true
+      uptime:
+        enabled: true
+      web:
+        enabled: true
+    export:
+      oraclecloud:
+        enabled: true
+        namespace: change-me # <3>
+#end::metrics[]
+---
+#tag::datasource[]
+datasources:
+  default:
+    url: jdbc:h2:mem:devDb;LOCK_TIMEOUT=10000;DB_CLOSE_ON_EXIT=FALSE
+    driverClassName: org.h2.Driver
+    username: sa
+    password: ''
+    schema-generate: CREATE_DROP
```

since we use flyway, this should be: 
```suggestion
    schema-generate: NONE
```